packages/search/config/schemas/admin.php

```diff
@@ -18,6 +18,7 @@
     'resourceId' => new Field\TextField('resourceId', searchable: false),
     'locale' => new Field\TextField('locale', searchable: false, filterable: true),
     'title' => new Field\TextField('title'),
+    'mediaId' => new Field\TextField('mediaId', searchable: false),
```

Comment:
```suggestion
    'mediaId' => new Field\IntegerField('mediaId'),
```

https://php-cmsig.github.io/search/schema/index.html#integerfield

the searchable: false is even not required when using a Int field its default value as integer fields are never searchable.